src/transform/constants.ts

```diff
@@ -0,0 +1,3 @@
+import {CssWhiteList} from './typings';
+
+export const forcedSanitizeWhiteList: CssWhiteList = {};
```

Comment:
This constant will contain some enforced rules from other plugins, if necessary.